tools/ete/ete_genetree_splitter.xml

```diff
@@ -0,0 +1,123 @@
+<tool id="ete_genetree_splitter" name="ETE GeneTree splitter" version="@VERSION@.1">
+    <description>from a genetree using the ETE Toolkit</description>
+    <macros>
+        <import>ete_macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <stdio>
+        <!-- Anything other than zero is an error -->
+        <exit_code range="1:" />
+    </stdio>
+    <command>
+    <![CDATA[
+python '$__tool_directory__/ete_genetree_splitter.py'
+--genetree '$genetreeFile'
+--gene_node $gene_node
+#if $gainlose_conditional.gainlose == "true"
+    --speciestree '$gainlose_conditional.speciesFile'
+    --species_format $gainlose_conditional.species_format
+    --gainlose
+#end if
+--output_format $output_format
+    ]]>
+    </command>
+    <inputs>
+        <param name="genetreeFile" type="data" format="nhx" label="GeneTree file" help="GeneTree in nhx format" />
+         <param name="gene_node" type="select" label="Select Gene node format" help="Select Gene node format from one of the option">
+            <option value="0" selected="true">gene_species</option>
+            <option value="1">species_gene</option>
+        </param>
+        <conditional name="gainlose_conditional">
+            <param name="gainlose" type="boolean" checked="false" label="Find out gene gain/lose" help="Find out gene gain/lose"/>
```

Comment:
Can you use a `type="select"` instead of boolean here? See https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#dynamic-options for a rationale.